src/Pages/FarmTImeLine.js

```diff
@@ -79,118 +106,144 @@ function FarmTimeline() {
+                            key={locationDataFromServer.code}
+                            value={locationDataFromServer.code}
+                          >
+                            {locationDataFromServer.displayStr}
+                          </MenuItem>
+                        );
+                      })}
+                    </Select>
+                  </FormControl>
+                </ListItem>
+
+                <ListItem key="2">
+                  <FormControl className={classes.formControl}>
+                    <InputLabel id="fId-label">Field ID</InputLabel>
+                    <Select
+                      id="fieldId"
+                      name="fieldId"
+                      value={fieldId}
+                      onChange={handleChange}
+                    >
+                      {fields.map((field) => {
+                        return (
+                          <MenuItem key={field.identifier} value={field.identifier}>
+                            {field.identifier}
+                          </MenuItem>
+                        );
+                      })}
+                    </Select>
+                  </FormControl>
+                </ListItem>
+
+                <ListItem>
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <Grid container justify="space-around">
+                      <KeyboardDatePicker
+                        disableToolbar
+                        variant="inline"
+                        format="MM/dd/yyyy"
+                        margin="normal"
+                        id="start-date"
+                        label="From"
+                        value={selectedDate}
+                        onChange={handleStartDateChange}
+                        KeyboardButtonProps={{
+                          "aria-label": "change date",
+                        }}
+                      />
+                    </Grid>
+                  </MuiPickersUtilsProvider>
+                </ListItem>
+
+                <ListItem>
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                    <Grid container justify="space-around">
+                      <KeyboardDatePicker
+                        disableToolbar
+                        variant="inline"
+                        format="MM/dd/yyyy"
+                        margin="normal"
+                        id="en-date"
+                        label="To"
+                        value={selectedEndDate}
+                        onChange={handleEndateChange}
+                        KeyboardButtonProps={{
+                          "aria-label": "change date",
+                        }}
+                      />
+                    </Grid>
+                  </MuiPickersUtilsProvider>
+                </ListItem>
+
+                <ListItem key="5">
+                  <Button variant="outlined" color="primary" onClick={handleCLick}>
+                    SHOW
+            </Button>
```

Comment:
Please check the button style used in other files and update, to keep the styling consistent.

Reply:
done , made similar to other filess